recipes/salad/stir-fry-cabbage.md

```diff
@@ -21,11 +21,17 @@ return Stir-Fry Cabbage
 
 ## Ingredients
 
-A wok, vegetable oil (olive oil x), 1/2 of a large white cabbage, 3 pieces of 50g of slim Scallion (Green onion), 2 pieces of large dried chili pepper, a base sauce, 5 or more cloves of garlic.
+* A large wok
+* Vegetable oil (olive oil x)
+* 1/2 of a large white cabbage
+* 3 pieces of 50g of slim Scallion (Green onion)
+* 2 pieces of large dried chili pepper
+* 5 or more cloves of garlic
+* Soy sauce, oyster sauce and sugar for making a base sauce
 
-* Base sauce needs to be prepared boiled with following ingredients
+- Base sauce needs to be prepared boiled with following ingredients
   * 4:5:1 ratio of Soy sauce, Oyster sauce and sugar.
-  * 5:5 ratio of Soy sauce and sugar for veggie friends.
+  * 1:1 ratio of Soy sauce and sugar for veggie friends.
```

Comment:
surely it's actually 9:1 @mattlub's comments were confusing

Reply:
I am not sure what which one you are talking about, one for veggies? because the one for non-veggies has 3 different sauces, soy sauce, oyster sauce and sugar. Do you mean, 9 (soysauce+ oyster sauce) :1 (sugar) ?